db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/river/HentPersistertLøser.kt

```diff
@@ -39,69 +35,45 @@ class HentPersistertLøser(rapidsConnection: RapidsConnection, private val repos
             )
         }
 
-    override fun onPacket(packet: JsonMessage, context: MessageContext) {
-        eventName = EventName.valueOf(packet[Key.EVENT_NAME.str].asText())
-        onBehov(packet)
+    override fun onBehov(packet: JsonMessage) {
     }
 
-    override fun onBehov(packet: JsonMessage) {
+    override fun onBehov(behov: Behov) {
         measureTimeMillis {
-            val forespoerselId = packet[Key.FORESPOERSEL_ID.str].asText()
-            val transactionId = packet[Key.UUID.str].asText()
-            val event = packet[Key.EVENT_NAME.str].asText()
             logger.info("Skal hente persistert inntektsmelding med forespørselId $forespoerselId")
-            sikkerLogger.info("Skal hente persistert inntektsmelding for pakke:\n${packet.toPretty()}")
```

Comment:
Jeg vil gjerne ha log-statements i både sikker og åpen logg, for å lettere kunne følge flyten uten å bytte mellom logger

Reply:
Dette er logget allerede i Løseren i onPacket.